country_by_country/processor.py

```diff
@@ -57,12 +57,12 @@ def process(self, pdf_filepath: str) -> dict:
 
         assets = {
             "pagefilter": {},
-            "text_table_extractors": {},
-            "img_table_extractors": {},
-            "table_cleaners": {},
+            "text_table_extractors": [],
```

Comment:
As I understand it, this change allows setting up evaluation pipelines where we can compare the results of various extractors or cleaners.

Reply:
I'd say this change 1) should make it easier to iterate over all the assets 2) allows to have a pipeline running the same method but with variations, for example unstructured/yolox and unstructured/detectron2.

For example, let's assume I want to run a pipeline containing 3 extractors camelot, unstructured/yolox and unstructured/detectron2.

**Before this PR, roughly:**
assets = {"img_table_extractors": {"camelot": {"tables": xxx}, ", "unstructured": {"tables": xxx}}

img_table_extractors is a dict.

Not possible to have 2 "unstructured" extractors!

**After this PR, roughly:**
assets = {"img_table_extractors": [{"type": "camelot", "tables": xxx} , {"type": "unstructured", "params": {"hi_res_model_name": "yolox"}, "tables": xxx}, {"type": "unstructured", "params": {"hi_res_model_name": "detectron2"}, "tables": xxx}]}

img_table_extractors is an array.

Possible to have as many unstructured extractors as we want!

Reply:
Awesome!